core/src/main/scala/org/http4s/Message.scala

```diff
@@ -366,10 +366,14 @@ sealed trait MaybeResponse[F[_]] {
 object MaybeResponse extends MaybeResponseInstances
 
 trait MaybeResponseInstances {
-  implicit def http4sMonoidForMaybeResponse[F[_]]: Monoid[MaybeResponse[F]] =
+
+  /** Intentionally not implicit. Do not use unless you've read and understood the
+    * deprecation warning on [[instance]].
+    */
+  def http4sMonoidForMaybeResponse[F[_]]: Monoid[MaybeResponse[F]] =
```

Comment:
I didn't leave behind a deprecated implicit version.  This all becomes obsolete with #1424.

And now I see that the scaladoc is a lie.